scss/_custom-forms.scss

```diff
@@ -23,35 +23,35 @@
       cursor: $cursor-disabled;
       background-color: $custom-select-disabled-bg;
 
-      &:after {
+      &::after {
         @extend .btn-primary:disabled;
       }
     }
   }
   .custom-file-label {
-    &:after {
+    &::after {
+
+      // make the button overlap the border
+      top: $input-border-width * -1;
```

Comment:
It's not about border here, it's to scale the top depending on the border.

Reply:
That thing was there before this PR, I don't know why it looks like its new, but well the border wont be shown at all if I understand because thats the exact size of the border turned to negative value

Reply:
Ok I get the purpose (and it's not new indeed it was just moved from the bottom of the file)